backend/core/src/listings/views/config.ts

```diff
@@ -15,9 +15,6 @@ const views: Views = {
     select: [
       "listings.id",
       "listings.name",
-      "listings.applicationDueDate",
-      "listings.applicationDueTime",
-      "listings.applicationOpenDate",
```

Comment:
Do we want to remove these as part of this change? (I wonder if a separate small PR to do this would be better?)